bitseq/store.go

```diff
@@ -70,46 +70,47 @@ func (h *Handle) Exists() bool {
 	return h.dbExists
 }
 
+// New method returns a handle based on the receiver handle
+func (h *Handle) New() datastore.KVObject {
+	h.Lock()
+	defer h.Unlock()
+
+	return &Handle{
+		app:   h.app,
+		id:    h.id,
+		store: h.store,
+	}
+}
+
+// Copy deep copies the handle into the passed destination object
+func (h *Handle) Copy(o datastore.KVObject) error {
```

Comment:
Both the `Copy` and `New` functions are not defined to my personal taste.
`New` operating on a package is different from returning an object of receiver type. Calling it `New` is making it less intuitive. I do understand that it is required to be a receiver type implementing `KVConstructor`. To make it more appropriate and intuitive, can you call this `MetadataClone()` which is exactly what the `New` represents here.

`Copy` again can be seen in the same vein as a package function with dst and src. Can you consider changing the name to say `CopyTo` which makes it different from common understanding of such `Copy` functions and easy to read ?